pkg/klusterlet/clusterinfo/manager.go

```diff
@@ -20,7 +21,7 @@ func (r *ClusterInfoReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	claimSource := clusterclaim.NewClusterClaimSource(r.ClaimInformer)
 	nodeSource := &nodeSource{nodeInformer: r.NodeInformer.Informer()}
 	return ctrl.NewControllerManagedBy(mgr).
-		WatchesRawSource(claimSource, &clusterclaim.ClusterClaimEventHandler{}).
+		WatchesRawSource(claimSource, clusterclaim.NewClusterClaimEventHandler(labels.Everything())).
```

Comment:
clusterinfo controller should consider all kinds of claims(including customer created claims that has no label)